shared/Mocks/StartupSocialTesting.cs

```diff
@@ -48,7 +48,9 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<AppSettings>(Configuration.GetSection("AppSettings"));
 
             //Sql client not available on mono
-            var useInMemoryStore = !_runtimeEnvironment.OperatingSystem.Equals("Windows", StringComparison.OrdinalIgnoreCase);
+            var useInMemoryStore = !_platform.IsRunningOnWindows
```

Comment:
Can't NanoServer connect to SQL? Or is this because MusicStore by default uses a LocalDB?